config/database.yml

```diff
@@ -24,6 +24,7 @@ default: &default
 development:
   <<: *default
   database: askdarcel_development
+  url: <%= ENV['DATABASE_URL'] %>
```

Comment:
So if I understand correctly, this line effectively does nothing?  At best, it's just there to make it clear that rails is looking at that env variable for db url?